lib/bimultimap.ex

```diff
@@ -643,11 +643,7 @@ defmodule BiMultiMap do
     import Inspect.Algebra
 
     def inspect(bimultimap, opts) do
-      concat([
-        "#BiMultiMap<",
-        Inspect.List.inspect(BiMultiMap.to_list(bimultimap), opts),
-        ">"
-      ])
+      concat(["BiMultiMap.new(", to_doc(BiMultiMap.to_list(bimultimap), opts), ")"])
```

Comment:
MapSet still uses `Inspect.List.inspect`: https://github.com/elixir-lang/elixir/blob/v1.14.0/lib/elixir/lib/map_set.ex#L506-L514

I don't know what's the difference, but I think we should still use old approach here to match Elixir's behaviour?

Reply:
hmm, the docs here https://hexdocs.pm/elixir/1.14/Inspect.html#inspect/2 say that `to_doc` should be used :P

Reply:
@mkaput bump

Reply:
Hmm, looking at the source code `to_doc` seems to be going to call `Inspect.List.inspect` anyway. Let's leave this as you wrote.